server/src/test/java/org/opensearch/index/translog/transfer/BlobStoreTransferServiceTests.java

```diff
@@ -169,9 +169,13 @@ public void testBuildTransferFileMetadata_NonEmptyInputStream() throws IOExcepti
     }
 
     public void testBuildTransferFileMetadata_InputStreamExceedsLimit() {
-        byte[] largeData = new byte[1025]; // 1025 bytes, exceeding the 1KB limit
+        byte[] largeData = new byte[10250]; // 10250 bytes, exceeding the 1KB limit
```

Comment:
any reason for changing this from 1025 to 10250? 1025 seems better since most of remote store providers have limit around 1024 for metadata.